tests/Unit/PointwiseFunctions/Hydro/Test_ComovingMagneticField.cpp

```diff
@@ -20,6 +20,14 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.Hydro.ComovingMagneticField",
   pypp::SetupLocalPythonEnvironment local_python_env(
       "PointwiseFunctions/Hydro/");
   const DataVector used_for_size(5);
+  pypp::check_with_random_values<1>(
+      static_cast<tnsr::A<DataVector, 3> (*)(
+          const tnsr::I<DataVector, 3>&, const tnsr::I<DataVector, 3>&,
+          const Scalar<DataVector>&, const Scalar<DataVector>&,
+          const tnsr::I<DataVector, 3>&, const Scalar<DataVector>&)>(
+          &comoving_magnetic_field<DataVector>),
+      "TestFunctions", "comoving_magnetic_field", {{{0.0, 1.0}}},
+      used_for_size);
```

Comment:
Can you test that `b^2=b^ab_a` from the 3 functions?

Reply:
done.